### ISSUE TITLE
The Agoda website has an issue with the app

### ISSUE THREAD

**Issue by williamadoss79** [travelpayouts/travel-app-ios#63]
The Agoda website has an issue with the app, it gets stuck on the loading page

https://user-images.githubusercontent.com/110046721/181061209-c59401bb-5e9b-4363-a3a7-76f65300294b.mp4

**Comment by tpsupport** [comment]
Hi! Could you please double-check if the problem still persists? Unfortunately, we failed to reproduce it. If it still gets stuck on the "Find the best deals" screen, could you please describe in more detail how we can reproduce it on our end? Specify the location, the hotel you've clicked on, dates, number of guests, etc. Also, please provide us the model of your iOS-device.

**Comment by williamadoss79** [comment]
Yes, it still gets stuck on the "Find the best deals" screen, I think Agoda website is not compatible with the app browser

Paris, August 10-11
Hotel Daumesnil - Vincennes 
iPhone 13 Pro iOS, v15.3
Travel App iOS 6.6.1

**Comment by tpsupport** [comment]
Hi @williamadoss79!

Sorry for the delay, on our end it doesn't get stuck on the "Find the best deals" screen when selecting an Agoda deal regardless of model and iOS version. Could you please check if the problem still persists? If so, please provide us with all the details with which we can certainly reproduce it on our side
